test/constructors.cpp

```diff
@@ -6,6 +6,10 @@
 #include <iterator>
 #include <algorithm>
 #include <vector>
+#include <string>
+#include <array>
+#include <string_view>
+#include <iostream>
```

Comment:
Format should have sorted these out.

Reply:
Oops, I was supposed to remove all the `<iostream>` includes.

Or did you want alphabetical header ordering? I don't mind doing that by-hand but sometimes, header order matters so I prefer to group things logically.

Reply:
I see from extraneous comments after braces that you are using clang-format, I would have expected it to arrange includes alphabetically per group. It was more of a comment on running clang-format on all the files than on sorting includes.